app/assets/javascripts/reducers/articles.js

```diff
@@ -8,6 +8,14 @@ const initialState = {
   sortKey: 'character_sum'
```

Comment:
This does not work to actually sort them initially.

I suggest instead to do a second dispatch in the fetchArticles action, right after RECEIVE_ARTICLES, to sort them by `character_sum`.